### DISCUSSION TITLE
Halloween is over, but not for Chtulu

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#15176] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Chtulu still uses the oooooold halloween pet sprite.
Someone should try and fix that *wink wink* *nudge nudge* 
https://github.com/FakeFishGames/Barotrauma/pull/13327

![image](https://github.com/user-attachments/assets/c24e06eb-64e9-471d-8c60-0501261d64c0)

### Reproduction steps

🎃 

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

**Comment by Regalis11** [comment]
This is not a bug, we intentionally kept this old sprite: it's supposed to be a sort of easter eggish monster anyway (a flying monster underwater?).

> Someone should try and fix that wink wink nudge nudge
https://github.com/FakeFishGames/Barotrauma/pull/13327

I was under the impression this PR wasn't done yet?
